tutorials/11-probabilistic-pca/11_probabilistic-pca.jmd

```diff
@@ -5,53 +5,53 @@ weave_options:
   error : false
 ---
 
-# Overview of PCA
+## Overview of PCA
 
-Principal component analysis (PCA) is a fundamental technique to analyse and visualise data. <span style="color:green">It is an unsupervised learning method mainly used for dimensionality reduction. </span>.
+Principal component analysis (PCA) is a fundamental technique to analyse and visualise data. It is an unsupervised learning method mainly used for dimensionality reduction.
 
-For example, we have a feature matrix $\mathbf{A} \in \mathbb{R}^{m \times n}$, and we would like to extract $d$ principal components which captures most of the information from the matrix but reside in low dimensions (i.e. $d<n$). The aim is to  understand $\mathbf{A}$ through a lower dimensional subspace (e.g. two-dimensional subspace where visualisation is convenient) spanned by the principal components. Geometrically, [singular value decomposition (SVD)](https://en.wikipedia.org/wiki/Singular_value_decomposition) decomposes the action of $\mathbf{A}$ into rotation-scaling-rotation.
+For example, we have a data matrix $\mathbf{X} \in \mathbb{R}^{N \times D}$, and we would like to extract $k << D$ principal components which captures most of the variation information from the original matrix but live in low dimensions. The goal is to understand $\mathbf{X}$ through a lower dimensional subspace (e.g. two-dimensional subspace for visualisation convenience) spanned by the principal components.
```

Comment:
```suggestion
For example, we have a data matrix $\mathbf{X} \in \mathbb{R}^{N \times D}$, and we would like to extract $k << D$ principal components which captures most of the information from the original matrix. The goal is to understand $\mathbf{X}$ through a lower dimensional subspace (e.g. two-dimensional subspace for visualisation convenience) spanned by the principal components.
```

"variation information" seems redundant here.
Just "information"?

Also the "live in low dimensions" is being supplanted by the "through a lower dimensional subspace" in the following sentence.

Reply:
Additionally, watch out that you are saying "spanned by the principal components" without indicating that the principal components are vectors.
A vector (sub)space can only be spanned by vectors.
However, I think it should be clear.

Reply:
@storopoli Many thanks for detailing these, so helpful to go through the detials. Yes, I changed many expressions from my previous version (many of which are borrowed from the references). 
1. `variation information`: like many regression tasks (classic  PCA is indeed a linear projection/regression), PCA aims to capture most of the variations/variances along the axes (the principal loading directions). I think `variation` is used in most texts, e.g. [Wikipedia](https://en.wikipedia.org/wiki/Principal_component_analysis) (apologize for using Wikipedia as a source of reference, but the contents are valid): 
"This is accomplished by linearly transforming the data into a new coordinate system where (most of) the variation in the data can be described with fewer dimensions than the initial data."
2. `low dimension`: there are two usages of "low dimensions": "live in low dimensions" and "through a lower dimensional subspace". I would keep both but definitely need to make it clear that spaces are spanned by vectors ("vector space" is commonly used in linear algebra; here we particularly refer to the principlal orthogonal loading directions along which the data variance is maximized). It would be safe to say 'spanned by the principal components' (e.g. a ref [here](https://arxiv.org/abs/1907.12159)).